Integrations/integration-opswat-metadefender-v2.yml

```diff
@@ -0,0 +1,342 @@
+    USE_CLOUD = PARAMS.get('cloud', False)
+    HIGH_THRESHOLD = int(PARAMS.get('highPercnt', 66))
+    LOW_THRESHOLD = int(PARAMS.get('lowPercnt', 34))
+
+
+
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    DEFAULT_HEADERS = {
+        'Accept': 'application/json'
+    }
+
+    HTTP_ERROR_CODES = {
+         '400': 'Request failed, got status 400: bad request. Check command parameters',
+         '401': 'Request failed, got status 401: unauthorized. Check your API Key',
+         '404': 'Request failed, got status 404: not found. Check integration URL Address',
+     }
+
+    ''' HELPER FUNCTIONS '''
+    def http_req(method='GET', url_suffix='', file=None, parse_json=True):
+        files = None
+        url = BASE_URL + url_suffix
+        headers = DEFAULT_HEADERS
+        if USE_CLOUD:
+            headers['apikey'] = API_KEY
+        if file:
+            headers['filename'] = file['file_name']
+            files = {file['file_name']:file['file']}
+        LOG(f'running request with url={url}\n\tfiles={files}')
+        if method.upper() == 'GET':
+            res = requests.get(url, verify=USE_SSL, headers=headers)
+        elif method.upper() == 'POST':
+            res = requests.post(url, verify=USE_SSL, files=files, headers=headers)
+        else:
+            return_error(f'Got unsupporthed http method: {method}')
+            return ''
+        status = res.status_code
+        if status != 200:
+            if str(status) in HTTP_ERROR_CODES:
+                return_error(HTTP_ERROR_CODES[str(status)])
+                return ''
+            else:
+                return_error(f'Request failed got status {status}')
+                return ''
+        if parse_json:
+            return res.json()
+        else:
+            return res.content
+
+    def scan_file(file_entry_id):
+        try:
+            file_entry = demisto.getFilePath(file_entry_id)
+        except ValueError as e:
+            return_error(f'Failed to find file entry with id:\"{file_entry_id}\". got error: {e}')
+            return ''
+        file_name = file_entry['name']
+        shutil.copy(file_entry['path'], file_name)
+        try:
+            with open(file_name, 'rb') as f:
+                res = http_req(method='POST',url_suffix='file',file={'file_name': file_name, 'file':f})
+        finally:
+            shutil.rmtree(file_name, ignore_errors=True)
+        return res, file_name
+
+
+    def get_hash_info(file_hash):
+        res = http_req(method='GET',url_suffix=f'hash/{file_hash}')
+        return res
+
+    def get_scan_result(scan_id):
+        res = http_req(method='GET',url_suffix=f'file/{scan_id}')
+        return res
+
+    def get_hash_info_command():
+        file_hash = demisto.args()['hash']
+        res = get_hash_info(file_hash)
+        if len(res) == 0:
+            return
```

Comment:
we dont want to return anything in this case? like no results found or some error ?